### PR TITLE
[Fix-16174] Incorrect cluster installation guide.

### LINKED ISSUE #16174: [Bug] [service down] service down and stop when use `bash bin/start-all.sh` a few seconds
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

I use cluster deploy to deploy dolphinscheduler service to 2 server.
server and service running env below:
os: centos7
db: postgresql-15.7
zookeeper: 3.7.2
python: python3.6
java: java-11

I deploy the service dolphinscheduler to 2 server and make them both be master-server and worker-server.
when use `bash bin/start-all.sh`, every service work well and run perfect, but a few second later, some master-server and worker-server went down, but some workflow still can run.

at the first time every server work well.
image

few seconds later, some server down but some workflow can still run.
image

how to solve it?  here is two server's master-server log file.
[dolphinscheduler-master.log](https://github.com/user-attachments/files/15883737/dolphinscheduler-master.log)
[dolphinscheduler-master_2.log](https://github.com/user-attachments/files/15883738/dolphinscheduler-master_2.log)

### What you expected to happen

service work well and not down.
[dolphinscheduler-master_2.log](https://github.com/user-attachments/files/15883753/dolphinscheduler-master_2.log)
[dolphinscheduler-master.log](https://github.com/user-attachments/files/15883754/dolphinscheduler-master.log)

### How to reproduce

use cluster deploy and make them all master-server and worker-server.then start the service.

### Anything else

_No response_

### Version

3.2.x

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

Fix #16174

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by ruanwenjun — DISMISSED** [review]
LGTM

File: docs/docs/zh/guide/installation/pseudo-cluster.md [hunk 1]
```diff
@@ -70,30 +70,7 @@ chmod 600 ~/.ssh/authorized_keys
...
-alertServer="localhost"
-apiServers="localhost"
-
-# DolphinScheduler installation path, it will auto-create if not exists
-installPath=~/dolphinscheduler
-
-# Deploy user, use the user you create in section **Configure machine SSH password-free login**
-deployUser="dolphinscheduler"
-```
+完成基础环境的准备后，需要根据你的机器环境修改配置文件。配置文件可以在目录 `bin/env` 中找到，他们分别是 `dolphinscheduler_env.sh`。
```
Comment (Radeity) [inline docs/docs/zh/guide/installation/pseudo-cluster.md]: If do not have other configuration file, maybe remove “分别是”?

Comment (SbloodyS) [reply docs/docs/zh/guide/installation/pseudo-cluster.md]: Thanks. Fixed.

**Review by Radeity — APPROVED** [review]
LGTM

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**